packages/build/package.json

```diff
@@ -53,6 +53,8 @@
   "license": "MIT",
   "dependencies": {
     "@bugsnag/js": "^7.0.0",
+    "@grpc/grpc-js": "^1.2.7",
```

Comment:
Uses `@grpc/grpc-js` instead of `grpc` to avoid adding native modules. 
Also, only `@grpc/grpc-js` supports Unix domain sockets.